sysutils/mcfly/Portfile

```diff
@@ -19,273 +19,287 @@ long_description    McFly replaces your default ctrl-r Bash history search \
                     executed commands. McFly's suggestions are prioritized \
                     in real time with a small neural network.
 
-checksums           mcfly-0.2.5.tar.gz \
-                    rmd160  6f787abdf5de81e0313606af194588c571845a97 \
-                    sha256  e90b12545b49d8cf3a79cb7cb03f0516833a3d78accb2cd75e1ef28b6a1ad058 \
-                    size    242571 \
-                    aho-corasick-0.6.6.crate \
-                    sha256  c1c6d463cbe7ed28720b5b489e7c083eeb8f90d08be2a0d6bb9e1ffea9ce1afa \
-                    rmd160  cce62fd1a33e14989e1f2f62b9a52ba10b9ad170 \
-                    size    26330 \
+checksums           mcfly-0.3.1.tar.gz \
+                    rmd160  c90e99eefee280a8d35c6913c167bff695d34ebb \
+                    sha256  c2831e270bf06d7557025bb7b5333cadc996c1ad9feca2e187a419046178eb79 \
+                    size    242536 \
                     ansi_term-0.11.0.crate \
                     sha256  ee49baf6cb617b853aa8d93bf420db2383fab46d314482ca2803b40d5fde979b \
                     rmd160  0bc10d826fc7a658ac1026dac333cc54f26f7c5b \
                     size    17087 \
+                    sha256  ee49baf6cb617b853aa8d93bf420db2383fab46d314482ca2803b40d5fde979b \
```

Comment:
You're duplicating all these sha256 lines. Why?

Reply:
Fixed.